qiita_ware/test/test_util.py

```diff
@@ -19,11 +19,8 @@
 from future.utils.six import StringIO, BytesIO
 
 from qiita_db.metadata_template import SampleTemplate, PrepTemplate
-from qiita_ware.util import (per_sample_sequences, template_to_dict,
-                             metadata_stats_from_sample_and_prep_templates,
-                             metadata_map_from_sample_and_prep_templates,
-                             stats_from_df, dataframe_from_template,
-                             open_file, _is_string_or_bytes)
+from qiita_ware.util import (per_sample_sequences, stats_from_df, open_file,
```

Comment:
This just says it changed because I removed so many util functions, and also their tests. All the remaining qiita_ware.util functions are in here for testing.